browser_history/generic.py

```diff
@@ -337,15 +356,41 @@ def to_csv(self):
                 writer.writerow(row)
             return output.getvalue()
 
-    def to_json(self, json_lines=False):
+    def to_json(self, json_lines:bool = False) -> str:
```

Comment:
Another small thing black complains of
```suggestion
    def to_json(self, json_lines: bool = False) -> str:
```